.github/workflows/list_reservations.yml

```
@@ -0,0 +1,44 @@
+        READ_PASSWORDS: ${{ secrets.LOGIN4 }},${{ secrets.LOGIN_RAYMOND_NANA }},${{ secrets.LOGIN_RAYMOND_NANA }},${{ secrets.LOGIN_ANTHONY }}
+    - uses: actions/upload-pages-artifact@v3
+      with:
+        path: upcoming_reservations
+    # This last step will create `steps.read_all.outputs.page_url` presumably
+
+
+  deploy:
+    needs: read_all
+    environment:
+      name: github-pages
+      url: ${{ steps.read_all.outputs.page_url }}
```

actionlint (1.7.12)

[error] property "read_all" is not defined in object type {deployment: {conclusion: string; outcome: string; outputs: {page_url: string}}}

(expression)